### ISSUE TITLE
could not specify interface

### ISSUE THREAD

**Issue by vzaliva** [andreikop/python-ws-discovery#21]
```
$ discover                                                         
ERROR:ws-discovery:no interface found; use one of: en0, eth0
```

My wifi card interface name is `wlp2s0`. How I can specify it?

```
 $ ip link                                                                        
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: wlp2s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DORMANT group default qlen 1000
    link/ether 9c:b6:d0:8a:6d:cb brd ff:ff:ff:ff:ff:ff

```

**Comment by Code2Mirabeau** [comment]
i have the same probleme...
i dont know why, but there is a restriction in the file "cmdline.py"

On line 18 this code is a is a nonsense:
`# an external interface is needed
# darwin has en0, linux has eth0 by default
external_ifaces = ("en0", "eth0")


def get_valid_interface():
    ifaces = netifaces.interfaces()
    valid = list(set(external_ifaces).intersection(ifaces))
    return valid[0] if len(valid) == 1 else None`

"ifaces = netifaces.interfaces()"   Give all the Network interface (the good ones)
But !
because of its 2 lines:
>    valid = list(set(external_ifaces).intersection(ifaces))
>    return valid[0] if len(valid) == 1 else None`
 you never get Wifi or eth0 on Modern Linux... (the name ifaces as changed)
'Author must have à look at the SystemD change about Network ifaces name :))

The only interface recognized as "good" are:
"external_ifaces = ("en0", "eth0")"
which is hyper restrictive and incomprehensible.

**Quick Version:** add your interface in this list. like this:
external_ifaces = ("en0", "eth0", "wlx74da381a9ebd")

For my part I modified this part of code to get all the interfaces, and exclude only "lo"
I put the choice to have a iface interface in the command line

**Comment by andreikop** [comment]
@Code2Mirabeau, I've merged #24. Do you have any problems now?
See also #22 and #23
